CONTRIBUTING.md

```diff
@@ -59,7 +125,19 @@ items in a sentence.
 See Scribbr's [article](https://www.scribbr.com/commas/oxford-comma/) on the Oxford
 comma for a detailed explanation and examples.
 
-## vCluster terms
+### Code blocks
+
+Use `<>` to indicate placeholders in code blocks. For example:
+
+```bash
+kubectl get pods <pod-name>
+```
+
```

Comment:
Do we want to add something about how to to make code blocks into easy copy/paste and encourage using variables? 

For example:

```
$POD_NAME = pod-name
kubectl get pods $POD_NAME
```

Reply:
:+1: I also added here docs support, which is useful in some scenarios.